src/main/target/system_stm32f4xx.c

```diff
@@ -358,6 +358,8 @@ uint32_t hse_value = HSE_VALUE;
  /* PLL_VCO = (HSE_VALUE or HSI_VALUE / PLL_M) * PLL_N */
 #if defined(COLIBRI)
  #define PLL_M      16
+#elif defined(ELLE0)
```

Comment:
@fvantienen , can you fix up using `TARGET_XTAL_MHZ` as enabled by PR https://github.com/betaflight/betaflight/pull/1737 ?